### DISCUSSION TITLE
Is there a way to view only a few frame of camera key frame transforms?

### DISCUSSION THREAD

**Discussion by ll-ghin** [keiwando/toonsquid#618] [Q&A]
I frequently do many camera movements in my animations.  For longer animations it becomes very confusing - see attached image.  Is there any way to basically ‘onion skin’ your camera moves?
![IMG_0846](https://github.com/keiwando/toonsquid/assets/20732395/06ab2ef6-b868-454d-9da5-9fcf244632b9)

**Comment by ll-ghin** [comment]
Alternatively - is there a way to manipulate camera movements to simulate impact / screen shakes?  I have tried using the 'bounce' feature on key frames to reduce the pure number of key frames I use, but I find it lacking for simulating the 'strong hit and camera shakes' I like to do in my animations.

**Comment by andydjames** [comment]
Davinci Resolve has a camera shake feature (part of the free version) that allows you to emulate a camera shake in post production.
https://www.youtube.com/watch?v=B3ya3MNMgLs [7mins in]
Please note, I only saw this on a video, I haven't used the app (yet!). There's a desktop and iPad version too, so there will be differences.

**Reply by ll-ghin** [reply]
Thank you Andy.  I was hoping to do it all in Toonsquid without having to use another software.  If no one else has a solution, I will try this out.

**Comment by andydjames** [comment]
At the moment there is no way of onion skinning keyframes.
However, if you select the camera layer, go to Properties, bump up Outside Darkening and enable "Always enabled", then you can more easily see in real time the camera movement.
I tried creating a black outline layer with the same proportions of the camera and creating different animation drawings to mimic its movement and then under hierarchy parent the camera layer to this "fake camera layer", but the camera layer doesn't work in the hierarchy tree.

**Reply by SharpsterDraws** [reply]
Could you copy and paste the key frames from the fake camera into the camera layer…. I think it would have to be done one key frame at a time so it would take a long time for a camera that moves a lot.
